README.md

```diff
@@ -1 +1,107 @@
 # avro-resolution_canonical_form
+
+This gem defines a [Resolution Canonical Form](#resolution_canonical_form) for Avro schemas.
+This is similar to the [Parsing Canonical Form](http://avro.apache.org/docs/1.8.1/spec.html#Parsing+Canonical+Form+for+Schemas)
+from the Apache Avro spec, but extends is to also include attributes that are
+relevant to schema resolution and compatibility.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'avro-resolution_canonical_form'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install avro-resolution_canonical_form
+
+## Resolution Canonical Form
+
+The Resolution Canonical Form extends the [Parsing Canonical Form](http://avro.apache.org/docs/1.8.1/spec.html#Parsing+Canonical+Form+for+Schemas)
+to include `default` and `aliases` attributes:
+
+* [ORDER] Order the appearance of fields in JSON objects as follows:
```

Comment:
I was a bit thrown by this bullet since the previous sentence talks about adding  `default` and `aliases` attributes and the next bullet talks about aliases but this bullet is about ordering? Is this just the ordering used in by the resolution canonical form?